operator/config/samples/karmada.yaml

```
@@ -34,36 +34,36 @@ spec:
       serviceSubnet: 10.96.0.0/12
     karmadaAggregatedAPIServer:
       imageRepository: docker.io/karmada/karmada-aggregated-apiserver
-      imageTag: v1.12.2
+      imageTag: v1.13.0
       replicas: 1
     karmadaControllerManager:
       imageRepository: docker.io/karmada/karmada-controller-manager
-      imageTag: v1.12.2
+      imageTag: v1.13.0
       replicas: 1
     karmadaScheduler:
       imageRepository: docker.io/karmada/karmada-scheduler
-      imageTag: v1.12.2
+      imageTag: v1.13.0
       replicas: 1
     karmadaWebhook:
       imageRepository: docker.io/karmada/karmada-webhook
-      imageTag: v1.12.2
+      imageTag: v1.13.0
       replicas: 1
     kubeControllerManager:
       imageRepository: registry.k8s.io/kube-controller-manager
       imageTag: v1.31.3
       replicas: 1
     karmadaMetricsAdapter:
       imageRepository: docker.io/karmada/karmada-metrics-adapter
-      imageTag: v1.12.2
+      imageTag: v1.13.0
       replicas: 2
-    # karmadaSearch: # the component `Karmadasearch` is not installed by default, if you need to install it, uncomment it and note the formatting
-      # imageRepository: docker.io/karmada/karmada-search
-      # imageTag: v1.12.2
-      # replicas: 1
-    # karmadaDescheduler: # the component `KarmadaDescheduler` is not installed by default, if you need to install it, uncomment it and note the formatting
-      # imageRepository: docker.io/karmada/karmada-descheduler
-      # imageTag: v1.12.2
-      # replicas: 1
+    karmadaSearch: # the component `Karmadasearch` is not installed by default, if you need to install it, uncomment it and note the formatting
+      imageRepository: docker.io/karmada/karmada-search
+      imageTag: v1.13.0
+      replicas: 1
+    karmadaDescheduler: # the component `KarmadaDescheduler` is not installed by default, if you need to install it, uncomment it and note the formatting
+      imageRepository: docker.io/karmada/karmada-descheduler
+      imageTag: v1.13.0
+      replicas: 1
```

@Pratham-B-Parlecha The code comments don't need to be removed.
```suggestion
    # karmadaSearch: # the component `Karmadasearch` is not installed by default, if you need to install it, uncomment it and note the formatting
      # imageRepository: docker.io/karmada/karmada-search
      # imageTag: v1.13.0
      # replicas: 1
    # karmadaDescheduler: # the component `KarmadaDescheduler` is not installed by default, if you need to install it, uncomment it and note the formatting
      # imageRepository: docker.io/karmada/karmada-descheduler
      # imageTag: v1.13.0
      # replicas: 1
```